docs/erp_bereitstellen.adoc

```diff
@@ -1240,6 +1234,8 @@ NOTE: Das signierte FHIR-Bundle wird als Ganzes gespeichert und steht inkl. der
 
 NOTE: Für den Versicherten wird eine Kopie des Bundles im JSON-Format inkl. serverseitiger Signatur bereitgestellt, die an der Stelle `<reference value="f8c2298f-7c00-4a68-af29-8a2862d55d43"/>` referenziert wird.
 
+NOTE: Für den Fall, dass das Clientsystem beim Aufruf der Operation keinen Response erhält, soll der Aufruf wiederholt werden. Wenn im Response ein Fehler 403 mit dem OperationOutcome "Task not in status draft but in status ready" zurückkommt, war der erste Versuch bereits erfolgreich. Wenn eine Response mit dem Status 200 zurückkommt, war der zweite Versuch erfolgreich.
```

Comment:
Dieser Fehler und die Beschreibung müssten noch in https://github.com/gematik/api-erp/blob/master/docs/erp_statuscodes.adoc beschrieben werden